README.md

```diff
@@ -39,6 +39,12 @@ Simple use cases:
 Comment.filter_by({ 'post_id' => '1' })
 # => WHERE post_id = 1
 
+Comment.filter_by({ 'post_id' => ['1', '2'] })
+# => WHERE post_id IN (1, 2)
+
+Comment.filter_by({ 'post_id_not' => '3' })
```

Comment:
the other change is that you now can append `_not` and exclude something

Reply:
will we still be able to handle when value is nil cases?